setup.py

```diff
@@ -108,10 +107,12 @@ def cythonize(extensions, **_ignore):
                 extension.sources[:] = sources
             return extensions
 
+    README = open('README.md', 'r').read()
```

Comment:
You'll need `long_description_content_type='text/markdown',` to get Markdown parsing on PyPI.

Reply:
Thanks.